tests/connector/cosmosis/test_cosmosis_module.py

```diff
@@ -49,12 +49,33 @@ def fixture_minimal_config() -> DataBlock:
     return result
 
 
+@pytest.fixture(name="config_with_derived_parameters")
+def fixture_config_with_derived_parameters() -> DataBlock:
+    result = DataBlock()
+    result.put_string(
+        option_section,
+        "Likelihood_source",
+        expandvars(
+            "${FIRECROWN_DIR}/tests/likelihood/lkdir" "/lk_derived_parameter.py"
+        ),
+    )
+    result.put_double("sample_parameters_sections", "derived_param0", 12.5)
```

Comment:
It is not necessary to insert this parameter into the DataBlock provided by the sampler. It will be inserted by Firecrown.